debian/changelog

```diff
@@ -0,0 +1,30 @@
+nutcracker (0.4.1+dfsg-1) unstable; urgency=medium
```

Comment:
Debian support is completely unrelated to new redis command support and should not be part of the same PR.

Additionally, I don't have any experience with debian packaging and wouldn't be able to review this, anyway - The debian package maintainers who originally wrote these files could continue to do so without these being checked in.

Reply:
yes. you may ignore all the debian folder